mrp_production_batch/views/mrp_production_views.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <menuitem
+        id="menu_mrp_production_batch"
+        name="MRP Batch Production"
+        parent="mrp.menu_mrp_manufacturing"
+        action="mrp_production_batch"
+        sequence="21"
+    />
+    <data>
+        <record id="view_mrp_production_tree_inherit" model="ir.ui.view">
```

Comment:
Don't forget the guidelines
` <record id="mrp_production_tree_view" model="ir.ui.view">`